cmd/resultserver/main.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright © 2019 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+)
+
+func defineFlags(cmd *cobra.Command) {
+	cmd.Flags().String("address", "127.0.0.1", "Server address")
+	cmd.Flags().String("port", "443", "Server port")
+	cmd.Flags().String("volume", "", "Volume name ??")
```

Comment:
Is this currently unused?

Reply:
yep, leftover from the original implementation.

Reply:
Fixed.